src/main/kotlin/no/nav/amt/arena/acl/utils/RowMapperExtensionFunctions.kt

```diff
@@ -42,3 +42,12 @@ fun ResultSet.getZonedDateTime(columnLabel: String): ZonedDateTime {
 	return getNullableZonedDateTime(columnLabel) ?: throw IllegalStateException("Expected $columnLabel not to be null")
 }
 
+fun ResultSet.getInteger(columnLabel: String): Int? {
+	val value = this.getInt(columnLabel)
+	return if (this.wasNull()) null else value
+}
+
+fun ResultSet.getNullableLong(columnLabel: String): Long? {
+	val value = this.getLong(columnLabel)
+	return if (this.wasNull()) null else value
+}
```

Comment:
Man trenger egentlig ikke å mappe om noe her, det holder å bruke `return this.getLong(columnLabel)`